src/accounts/ethereum.ts

```diff
@@ -106,8 +112,9 @@ export function NewAccount(derivationPath = "m/44'/60'/0'/0/0"): { account: ETHA
 
 export async function GetAccountFromProvider(provider: ethers.providers.ExternalProvider) {
     const ETHprovider = new ethers.providers.Web3Provider(provider);
-    const signer = ETHprovider.getSigner();
-    const address = await signer.getAddress();
+    const metamask = new Metamask(ETHprovider);
```

Comment:
Only Metamask is supported here, but `ProviderNames` also supports Ledger. Why ?

Reply:
I've removed the Ledger integration from this PR, to focus on Software wallets. The `BaseProviderWalet` signature should cover abstraction for both Hardware and Software wallets.

To avoid any confusion I've removed mentions to Ledger in this PR